app/views/devise/sessions/new.html.haml

```diff
@@ -12,5 +12,6 @@
     = f.input_field :password, autocomplete: 'off', class: "form-control"
 
   .form-group
-    = f.button :submit, t('.sign_in'), class: "button"
+    = govuk_button() { t('.sign_in') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)